.gitignore

```diff
@@ -0,0 +1,19 @@
+# Ignore vi swapfiles
+*.swp
+.*.swp
+
```

Comment:
Not that this is _super_ important, but I've always gone by the principle recommended in the [Git documentation](https://git-scm.com/docs/gitignore) which is that editor-specific ignore rules, or anything else that will only be relevant to some developers' workflows, should go in a local ignore file like `$HOME/.config/git/ignore` (or whatever you set `core.excludesFile` to be). So I would suggest removing these lines.

```suggestion
```

Reply:
Yep, I think a lot has changed with `git` too over the years.  I've seen this done in other places, particularly with other editors (`.idea` comes to mind with those that use IntelliJ-derived development environments) or other operating systems (`.DS_Store` on MacOS X).

I learned to use `git` back in 2008 when things were a lot more primitive, so I've probably learned bad habits here. :-)  Good to learn there is a better, less viral, way of doing things.